modules/apps/style-book/style-book-test/src/testIntegration/java/com/liferay/style/book/service/test/StyleBookEntryLocalServiceTest.java

```diff
@@ -99,6 +99,26 @@ public void testDeleteStyleBookEntryByExternalReferenceCode()
 				styleBookEntry.getStyleBookEntryId()));
 	}
 
+	@Test
+	public void testFetchAndGetStyleBookEntryByExternalReferenceCode()
+		throws Exception {
+
+		String externalReferenceCode = RandomTestUtil.randomString();
+
+		_styleBookEntryLocalService.addStyleBookEntry(
+			externalReferenceCode, TestPropsValues.getUserId(),
+			_group.getGroupId(), false, null, RandomTestUtil.randomString(),
+			null, RandomTestUtil.randomString(), _serviceContext);
+
+		Assert.assertNotNull(
+			_styleBookEntryLocalService.
+				fetchStyleBookEntryByExternalReferenceCode(
+					externalReferenceCode, _group.getGroupId()));
+
+		_styleBookEntryLocalService.getStyleBookEntryByExternalReferenceCode(
+			externalReferenceCode, _group.getGroupId());
```

Comment:
@lfbesada  You forgot to assert that the `StyleBookEntry` is not null.

Also, since we have both **get** and **fech**, please assert that an exception is thrown in case there is no StyleBookEntry and `getStyleBookEntryByExternalReferenceCode` is called. And null is returned in case `fetchStyleBookEntryByExternalReferenceCode` is called

Reply:
She did in that way because Brian asked for it in other prs

Reply:
@ealonso what did he say? Just "hey just do it because I am asking to"?

Reply:
I'm trying to find the commit, to clarify why we are not assert for null